manifesto.md

```diff
@@ -69,3 +69,7 @@ If we're giving out puppies, we're giving them to the people who've been with us
 
 ### No forced updates
 I hate it when I love the way an app is now and then it forces me to update to a new version where they have ruined the experience by trying to get the app to do too much and bloated the UI. _Don't you?_ Our plan is to keep things modular enough that if you :heart: the way the app looks now, you can keep it just the way you want it but still access new features.
+
+### No 'exit strategy'
+[@nelsonic](https://github.com/nelsonic) wrote an _excellent_ summary of why we
```

Comment:
@iteles I _disagree_ that the contents of the issue is "_excellent_" it's just a _brief_ response to the question. 👍